### PR TITLE
From a TODO in #1211. Add documentation for contributors.

### PR DESCRIPTION
**Pull request by adrian-rivera**
This changes the README file and adds a new section `Contributors`.
This section includes instructions to remind contributors to make an
update to the changelog.

This relates to #1211

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -665,6 +665,22 @@ bash assets/create.sh
...
+## Contributors
+
+Please don't forget to update `CHANGELOG.md` and describe your changes, this makes the release of a new `bat` version much easier.
+
+Each entry should follow the next format:
+- Short description of the change (one line)
+- Number of the pull request
+- Github's username
+
+See the next examples taken from `CHANGELOG.md`
```
Comment (keith-hall) [inline README.md:677]:
```suggestion
See the following examples taken from `CHANGELOG.md`
```
Note, I'm not sure we really need to call out some real examples from the changelog here, the instructions we post on PRs is (IMO) quite concise and clear:

> Could you please add an entry to the "unreleased" section in CHANGELOG.md? The format for entries is:
> `- Description what has been changed, see #123 (@user)`
> (where `#123` links to the bug ticket and/or the PR and user is your username.)

File: README.md [hunk 2]
```diff
@@ -665,6 +665,22 @@ bash assets/create.sh
 cargo install --path . --locked --force
 ```
 
+## Contributors
+
+Please don't forget to update `CHANGELOG.md` and describe your changes, this makes the release of a new `bat` version much easier.
```
Comment (keith-hall) [inline README.md:670]:
```suggestion
Please don't forget to update `CHANGELOG.md` and describe your changes (in the "unreleased" section), as this makes the release of a new `bat` version much easier.
```

File: README.md [hunk 3]
```diff
@@ -665,6 +665,22 @@ bash assets/create.sh
...
+
+Each entry should follow the next format:
```
Comment (keith-hall) [inline README.md:672]:
```suggestion
Each entry should follow the following format:
```

**Review by keith-hall — CHANGES_REQUESTED** [review]
Thanks for your contribution.
As per the comment at https://github.com/sharkdp/bat/issues/1211#issuecomment-706392560, I think we ideally want this info in a separate `CONTRIBUTING.md` file, which could be linked to from the readme.

**Comment by eth-p** [comment]
Although there isn't really any standard for this, I think `CONTRIBUTING.MD` is used more often than `CONTRIBUTORS.MD`.

For example, this is what Hacktoberfest has to say:

> ### Do: Follow a repository's contribution guidelines
> 
> Almost every repo looking for help has a `CONTRIBUTING.md` file in it, read it first!

**Comment by adrian-rivera** [comment]
> I feel like the file should be renamed. Although there isn't really any standard for this, I think `CONTRIBUTING.MD` is used more often than `CONTRIBUTORS.MD`.
> 
> For example, this is what Hacktoberfest has to say:
> 
> > ### Do: Follow a repository's contribution guidelines
> > Almost every repo looking for help has a `CONTRIBUTING.md` file in it, read it first!

Yep, I think you are right, `CONTRIBUTING` seems to be widely use, I'll do the change

**Review by sharkdp — APPROVED** [review]
Thank you for starting this @adrian-rivera. I added a few more sections to the `CONTRIBUTING.md` file.

@eth-p @keith-hall Let me know if you think we should add anything else to the document.

**Review by eth-p — APPROVED** [review]
Looks good to me!

**Pull request merged**